src/components/Recommend/Recommend.style.tsx

```diff
@@ -111,6 +111,14 @@ export const RecommendationBody = styled(MarkdownContent)`
   strong {
     color: black;
   }
+
+  a {
```

Comment:
Recommendations also uses markdown, but we don't want the new blues implementing on the location page yet. this reverts recommendations markdown links back to the original blue